docs/en/unity/integration.md

```diff
@@ -1,6 +1,10 @@
 # Integration Guide
 
-This page shows you how to download, important, and configure the Yodo1 Suit SDK in Unity.
+Yodo1 Suit SDK is an API wrapper for Yodo1's publishing services/business. It provides a comprehensive set of tools and APIs to effectively integrate Yodo1's services into your game development process. By utilizing Yodo1 Suit SDK, developers can easily implement features such as in-app purchases, ad monetization, analytics, anti-addition(China version COPPA) and more, improving your games and increasing revenue streams.
+
+With Yodo1 Suit SDK, developers can focus on creating the best possible gaming experience for their users, while relying on a robust set of tools and services to quickly and efficiently bring their games to market. Whether you're looking to launch on iOS or Android, Yodo1 Suit SDK is designed to simplify the integration process and help developers achieve their goals effectively and efficiently.
+
+And this page shows you how to download and set up Yodo1 Suit SDK in Unity.
 
 ## Getting started
```

Comment:
I actually suggest we move the SDK init here. Because the init method is very simple(the doc write like this will be head heavy and feet light), the most important thing for the developer is to go through the configuration steps.